packages/veritone-react-common/src/components/MediaInfoPanel/EditMetadataDialog/index.js

```diff
@@ -68,13 +70,15 @@ class EditMetadataDialog extends Component {
       metadataToUpdate.veritoneCustom = {};
       metadataToUpdate.veritoneCustom.source = this.state.source;
     }
-    if (this.isProgramLiveImageChanged()) {
+    if (this.state.programLiveImageFile) {
       metadataToUpdate.veritoneProgram = {};
-      metadataToUpdate.veritoneProgram.programLiveImage = this.state.programLiveImage;
+      metadataToUpdate.veritoneProgram.programLiveImage = this.state.programLiveImageFile;
     }
-    if (this.isProgramImageChanged()) {
-      metadataToUpdate.veritoneProgram = {};
-      metadataToUpdate.veritoneProgram.programImage = this.state.programImage;
+    if (this.state.programImageFile) {
+      if (!metadataToUpdate.veritoneProgram) {
+        metadataToUpdate.veritoneProgram = {};
+      }
+      metadataToUpdate.veritoneProgram.programImage = this.state.programImageFile;
```

Comment:
I think you can reduce this to a one liner /w https://lodash.com/docs/4.17.5#setWith

`setWith(metadataToUpdate, 'veritoneProgram', this.state.programImageFile, Object);`

Reply:
```javascript
mediaToUpdate = {
  ...mediaToUpdate,
  veritoneProgram: {
    programImage: this.state.programImageFile
  }
}
```